src/data/Pager/PagerTheme.ts

```diff
@@ -19,7 +21,6 @@ export interface PagerTheme {
 
 const baseTheme: PagerTheme = {
   base: 'items-center flex user-select-none',
-  arrows: 'h-8 w-8',
```

Comment:
Did u mean to delete this?

Reply:
Yea - it's not being used anywhere.. and isn't even included in PagerTheme type. Meant to add a comment about it.